compute_tools/README.md

```diff
@@ -40,21 +40,26 @@ compute moves between states.
 ```mermaid
 %% https://mermaid.js.org/syntax/stateDiagram.html
 stateDiagram-v2
+  terminating: TerminationPendingFast TerminationPendingImmediate
   [*] --> Empty : Compute spawned
   Empty --> ConfigurationPending : Waiting for compute spec
   ConfigurationPending --> Configuration : Received compute spec
   Configuration --> Failed : Failed to configure the compute
   Configuration --> Running : Compute has been configured
   Empty --> Init : Compute spec is immediately available
-  Empty --> TerminationPending : Requested termination
+  Empty --> terminating : Requested termination
   Init --> Failed : Failed to start Postgres
   Init --> Running : Started Postgres
-  Running --> TerminationPending : Requested termination
-  TerminationPending --> Terminated : Terminated compute
+  Running --> terminating : Requested termination
+  terminating --> Terminated : Terminated compute
   Failed --> [*] : Compute exited
   Terminated --> [*] : Compute exited
 ```
 
+> Note: TerminationPending is actually two values in ComputeStatus:
+> TerminationPendingFast and TerminationPendingImmediate, to reflect
+> whether we should wait 30s before returning from /terminate
```

Comment:
I'd like to see this moved into the diagram:

```
TerminationPendingFast --> Terminated : Terminated compute after giving control plane time to inspect the compute
TermiantionPendingImmediate --> Terminated : Terminated compute without giving control plane time to inspect the compute
```

What do you think? It keeps the diagram self-contained.